buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec

```diff
@@ -103,6 +104,7 @@ Requires: python3-matplotlib
 %{?fedora:Requires: python3-qtconsole}
 Requires: python3-PyYAML
 Requires: python3-mock
+Requires: python3-psutil
```

Comment:
This needs to be wrapped in a fedora check like `python3-qtconsole`

Reply:
Just out of curiosity how come some of the other `python3` packages are not wrapped?